lib/pb.ml

```diff
@@ -22,7 +22,7 @@ module KeyMap = MoreLabels.Map.Make(struct
     type t = key
     let compare a b =
       match UInt64.compare a.field_number b.field_number with
-      | 0 -> Pervasives.compare a.wire_type b.wire_type
+      | 0 -> compare a.wire_type b.wire_type
```

Comment:
Shall we add `Wire_type.compare` and use it here?

Reply:
Good idea. I'll move that entire `Map` there.